README.md

```diff
@@ -35,6 +36,38 @@ required for testing citus.
 * [Advanced fab Usage](#advanced-fab)
   * [Using Multiple Citus Installations](#multiple-installs)
 
+## <a name="running-automated-tests"></a>Running Automated Tests
+
+**Depending of the tests you trigger here, you can block at most 3 jobs slots in circleci for around 3 hours. Choose wisely the time you want to run the tests to not block development**
+
+If you want, you can run trigger a job which can run pgbench, scale and tpch tests. What the job does is:
+
+* It creates a cluster with the test resource group name
+* It connects to the coordinator
+* It runs the corresponding test for the job
+* It deletes the cluster.
+
+There is a separate job for each test and you can run any combinations of them. To trigger a job, you should create a branch which has specific prefixes.
+
+If the branch has a prefix pgbench, then pgbench job will be triggered.
+If the branch has a prefix scale, then scale job will be triggered.
+If the branch has a prefix tpch, then tpch job will be triggered.
+If the branch has a prefix all_test, then all jobs will be triggered.
+
+Each job uses a specific resource group name so that there will be at most 3 resource groups for these jobs. If there is already a resource group, then you should make sure that:
+
+* Someone else is currently not running the same test as you
+
+If not, then you can delete the resource group name from portal, you can find it by search the prefix `citusbot`. Under normal circumstances the resource group will already be deleted at the end of the test
+even if it fails.
+
+You can find your test results in https://github.com/citusdata/release-test-results.
+By default the tests will be run against `enterprise-master` and the latest released version. If you want to test on a custom branch you should change the config files of relevant tests with your custom branch name in:
+
+```text
```

Comment:
I think we can improve this readme by improving this  section by:

(a) refer to place in the README where document how to change the cluster size/node count etc
(b)  refer to place in the README where document how to change the pgbench/scale tests params
(c) Very briefly explain what should user in the branch do in order to change the above